tasks/install-from-source.yml

```diff
@@ -1,27 +1,41 @@
 ---
+- name: Ensure dependency list is set (RedHat)
+  set_fact:
+    git_package_deps:
```

Comment:
Changes have been long complete, could you provide some feedback to reach approval, where required?